Code/GraphMol/Depictor/Wrap/rdDepictor.cpp

```diff
@@ -142,11 +142,50 @@ void GenerateDepictionMatching3DStructure(RDKit::ROMol &mol,
   RDDepict::generateDepictionMatching3DStructure(
       mol, reference, confId, referencePattern, acceptFailure, forceRDKit);
 }
+
+bool isCoordGenSupportAvailable() {
+#ifdef RDK_BUILD_COORDGEN_SUPPORT
+  return true;
+#else
+  return false;
+#endif
+}
+
 void setPreferCoordGen(bool value) {
 #ifdef RDK_BUILD_COORDGEN_SUPPORT
   RDDepict::preferCoordGen = value;
 #endif
 }
+bool getPreferCoordGen() {
+#ifdef RDK_BUILD_COORDGEN_SUPPORT
+  return RDDepict::preferCoordGen;
+#else
+  return false;
+#endif
+}
+
+class UsingCoordGen : public boost::noncopyable {
```

Comment:
Do we need or want a RAII class for C++ use?

Reply:
Do we? All the usage of `setPreferCoordGen(true)` I have seen is from Python, never from C++ (neither in RDKit nor in our code).

Reply:
It's worth thinking about introducing some RAII classes for this as well as the other global configuration options (like the ones connected with non-tetrahedral stereo and legacy stereo).

But that would be a different PR